etc/smashbox.conf.template

```diff
@@ -136,3 +136,10 @@ share_sets = 1
 #
 # scp port to be used in scp commands, used primarily when copying over the server log file
 scp_port=22
+
+#
+# properties for defining directory depth and directory layout
+#
+dir_depth = 5
```

Comment:
are these values even used, I checked out your branch, without doing the config changes and they still pass.
The reason is, that the values from the testsets are put into the config on `run_test` (`bin/smash`):
```py
      def run_test(ts,i=0):
         if ts:
            for opt,val in zip(ts.keys(),ts.values()):
               setattr(config,opt,val)
               ...
```

So I think we can leave the config change out.